src/pages/PaymentPage.tsx

```diff
@@ -0,0 +1,211 @@
+import { AddCardStep } from '@/components/PaymentModal/AddCardStep'
+import { CardListStep } from '@/components/PaymentModal/CardListStep'
+import { CompletedAddCardStep } from '@/components/PaymentModal/CompletedAddCardStep'
```

Comment:
```suggestion
import { AddCardStep, CardListStep, CompletedAddCardStep } from '@/components'
```

아예 이렇게 불러오면 어떨까 싶기도 하네요!

Reply:
[e1e1a1b](https://github.com/next-step/react-payments/pull/134/commits/e1e1a1b2977f010ff0c59c683b8c1e3a591913b3)에서 베럴 파일을 생성하여 import 하도록 수정하였습니다!